src/components/Nav.js

```diff
@@ -15,19 +22,70 @@ const Nav = () => {
     <Box
       direction="row"
       justify="between"
-      align="center"
+      align={['xsmall', 'small'].includes(size) ? 'center' : 'start'}
       width="xlarge"
       alignSelf="center"
       gap="medium"
     >
-      <RoutedAnchor
-        path="/"
-        icon={<GrommetIcon size="large" color="plain" />}
-        label={size !== 'small' && <Text size="xlarge">grommet</Text>}
-      />
+      <Box justify="between" align="start" alignSelf="center">
+        <RoutedAnchor
+          path="/"
+          icon={<GrommetIcon size="large" color="plain" />}
+          label={
+            !['xsmall', 'small'].includes(size) && (
+              <Text size="xlarge">grommet</Text>
```

Comment:
It feels like this should always be the grommet theme font and color because it's more of a branded element -- can we explore wrapping this Text element in `ThemeContext.Extend` where the value would include the font from grommet theme?
```
global: {
    font: {
      family: `-apple-system,
         BlinkMacSystemFont, 
         "Segoe UI", 
         Roboto, 
         Oxygen, 
         Ubuntu, 
         Cantarell, 
         "Fira Sans", 
         "Droid Sans",  
         "Helvetica Neue", 
         Arial, sans-serif,  
         "Apple Color Emoji", 
         "Segoe UI Emoji", 
         "Segoe UI Symbol"`,
    },
  },
```

https://user-images.githubusercontent.com/12522275/179637751-d1ced39b-16d6-4018-bd72-5c17925e9a21.mov